changelog/pending/20231129--pkg-testing--extends-tests-to-cover-cases-in-issue-9054.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
```

Comment:
Internal test changes don't need changelogs, we just use these for user facing changes.